homeassistant/components/ambient_network/sensor.py

```diff
@@ -0,0 +1,310 @@
+        device_class=SensorDeviceClass.AQI,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+    ),
+    SensorEntityDescription(
+        key=TYPE_AQI_PM25_24H,
+        translation_key="pm25_aqi_24h_average",
+        device_class=SensorDeviceClass.AQI,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=0,
+    ),
+    SensorEntityDescription(
+        key=TYPE_BAROMABSIN,
+        translation_key="absolute_pressure",
+        native_unit_of_measurement=UnitOfPressure.INHG,
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_BAROMRELIN,
+        translation_key="relative_pressure",
+        native_unit_of_measurement=UnitOfPressure.INHG,
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_CO2,
+        native_unit_of_measurement=CONCENTRATION_PARTS_PER_MILLION,
+        device_class=SensorDeviceClass.CO2,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_DAILYRAININ,
+        translation_key="daily_rain",
+        native_unit_of_measurement=UnitOfPrecipitationDepth.INCHES,
+        device_class=SensorDeviceClass.PRECIPITATION,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_DEWPOINT,
+        translation_key="dew_point",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_FEELSLIKE,
+        translation_key="feels_like",
+        native_unit_of_measurement=UnitOfTemperature.FAHRENHEIT,
+        device_class=SensorDeviceClass.TEMPERATURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_HOURLYRAININ,
+        translation_key="hourly_rain",
+        native_unit_of_measurement=UnitOfVolumetricFlux.INCHES_PER_HOUR,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.PRECIPITATION_INTENSITY,
+        suggested_display_precision=2,
+    ),
+    SensorEntityDescription(
+        key=TYPE_HUMIDITY,
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.HUMIDITY,
+        state_class=SensorStateClass.MEASUREMENT,
+        suggested_display_precision=1,
+    ),
+    SensorEntityDescription(
+        key=TYPE_LASTRAIN,
+        translation_key="last_rain",
+        icon="mdi:water",
```

Comment:
You can now make use of icon translations: https://developers.home-assistant.io/blog/2024/01/19/icon-translations

Reply:
I did add the `icons.json` file but, somehow, I cannot make the icons show up in my dev environment. Looks like the icons never get loaded. Not sure what I'm missing.

Reply:
This requires a new frontend version that's not available yet. So we'll have to do a manual check if it works :)

Reply:
That was quick 😄